src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -54,3 +54,9 @@
 "Value"=dword:00000001
 "Title"="Enable enhanced Razor LSP server (requires restart)"
 "PreviewPaneChannels"="IntPreview,int.main"
+
+[$RootKey$\FeatureFlags\Razor\LSP\ShowAllCSharpCodeActions]
+"Description"="Show all C# code actions in Razor files. Note: This is experimental, some actions may not apply correctly, or at all!"
+"Value"=dword:00000000
+"Title"="Show all C# code actions in Razor files (requires restart)"
+"PreviewPaneChannels"="IntPreview,int.main"
```

Comment:
❓ is there a reason not to allow this in public preview as well?

Reply:
I would argue that we shouldn't push a random user feature without having at least a broader discussion, even under a preview flag.

Reply:
I guess I cheated by not having the discussion, because my opinion in such a discussion is that this is not something we want to allow people to turn on. I see this as an internal dogfooding feature so as people use the product they can help us identify which code actions they find most useful, which seem to work most reliably etc.